types/react/index.d.ts

```diff
@@ -355,7 +356,7 @@ declare namespace React {
         displayName?: string;
     }
 
-    interface ComponentClass<P = {}, S = ComponentState> extends StaticLifecycle<P, S> {
```

Comment:
why not making `type ComponentState = any` to have consistent type ( turned off type check for state ) everywhere where `ComponentState` is used ?

Reply:
I wanted to do so, but did not have the gut haha. I can definitely do that if `any` type for `CompnentState` is not too bad.